### DISCUSSION TITLE
Cloudflare CDN 加速指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#28] [Announcements]
# Cloudflare CDN 加速指南

最近被相关问题困扰，亲自尝试了一款名为 Cloudflare Speed Test 的小工具，非常好用。

https://zexwoo.blog/posts/tutorials/boost-cloudflare/

**Comment by Viderfate** [comment]
这个只是改善了你自己的访问速度，对于其他访问者来说，国内的访问依然很慢，而且你不可能让每个访问者都去修改自己电脑的Host文件，也不现实。Cloudflare的CDN国外访问速度飞起，但是国内访问真的慢的要死，无奈我也用的是Cloudflare，人家CDN免费，而且安全措施做的也好，比国内的阿里，腾讯好太多了，关键国内的CDN服务都要去备案才可以使用，这也是我没选国内的原因。楼主如果有更好的改善Cloudflare CDN国内访问速度的方法，希望可以分享，谢谢。

**Comment by zyhsck** [comment]
改完之后直接访问不了了...
